src/utils/common.c

```diff
@@ -1302,3 +1302,13 @@ void forced_memzero(void *ptr, size_t len)
 	if (len)
 		forced_memzero_val = ((u8 *) ptr)[0];
 }
+
+#ifdef CONFIG_ZEPHYR
+#include <zephyr/net/net_ip.h>
+extern char *inet_ntoa(struct in_addr in)
+{
+	char addr[NET_IPV4_ADDR_LEN];
```

Comment:
Shouldn't this be `static`, given the pointer is returned?

Reply:
Yes, good point.